src/GitHub.App/ViewModels/PullRequestFileNode.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using GitHub.Models;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestFileNode : IPullRequestFileNode
+    {
+        public PullRequestFileNode(string fullPath, PullRequestFileStatus status)
+        {
+            FileName = System.IO.Path.GetFileName(fullPath);
+            Path = System.IO.Path.GetDirectoryName(fullPath);
+            Status = status;
+        }
+
+        public string FileName { get; }
+        public string Path { get; }
```

Comment:
I tend to find property names like `Path` confusing. Path to what? How about `DirectoryName` or `DirectoryPath`? Bonus, that'll let you use `Path.GetFileName` rather than the fully qualified calls. :smile: